docs/framework/react/reference/usePrefetchInfiniteQuery.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
you need to add those to the `docs/config.json` file in order for them to show up in the sidebar

Reply:
Done!